### ISSUE TITLE
Calculation Groups: cannot create "Ordinal" column for sorting

### ISSUE THREAD

**Issue by zenzeinet** [otykier/TabularEditor#361]
Hi,
For Calculation Groups, I cannot find a way through TE UI to add a second column to the calculation group to specify the order of the items. This is the Ordinal column that is mentioned in the MS documentation, and needs to be of integer type. Without this column, the ordinal property of the Calculation Items doesn't do anything.
Using SSDT I can add a second column to the Calculation Groups, that is automatically set to Integer type.
BR
P.S.: Using SSAS 2019 , TE 2.8.6 , on a live connection to SSAS.

**Comment by otykier** [comment]
Until I fix the UI, the following script should work as a workaround to add the column:

```csharp
var column = Selected.Table.AddDataColumn("Ordinal");
column.DataType = DataType.Int64;
```

Execute the script while you have the calculation group selected in the Explorer Tree.

**Comment by zenzeinet** [comment]
Hi,
I can execute that, but I cannot save it to the server. It complains that a calculation group must have 2 columns, one source to Name and the other to Ordinal.
I can source Ordinal to Ordinal, but cannot find how to source the main one to Name. Maybe the creation of a calculation group from TE does not set the source column to Name? I haven't been able to set it, but I'm a newbie of the Advanced Editor (and cannot see that property through the GUI)
BR

**Comment by otykier** [comment]
I’ll look into it. Thanks!

**Comment by otykier** [comment]
Fixed in [2.9.0](https://github.com/otykier/TabularEditor/releases/tag/2.9.0).

**Comment by zenzeinet** [comment]
Great (this and all the other bugfixes). Upgrading to 2.9.0. You rock, thanks!
